Floofbot/Services/NicknameAlertService.cs

```diff
@@ -0,0 +1,174 @@
+                        builder.Description = $"You have recieved a warning in {badUser.Guild.Name}";
+                        builder.AddField("Reason", "Warned by BOT for an inappropriate name");
+                        builder.Color = Color.DarkOrange;
+                        await badUser.SendMessageAsync("", false, builder.Build());
+
+                        await channel.SendMessageAsync($"Got it! I warned {badUser.Username}#{badUser.Discriminator}!");
+                    }
+                    catch (Exception ex)
+                    {
+                        await channel.SendMessageAsync("Unable to warn user. Do I have the permissions?");
+                        Log.Error("Unable to warn user for bad name: " + ex);
+                    }
+                    messageDic.Remove(msg.Id);
+                    return;
+                }
+                if (reaction.Emote.Name.Equals(kickEmoji.Name))
+                {
+                    try
+                    {
+                        //sends message to user
+                        EmbedBuilder builder = new EmbedBuilder();
+                        builder.Title = "🥾 Kick Notification";
+                        builder.Description = $"You have been Kicked from {badUser.Guild.Name}";
+                        builder.AddField("Reason", "Kicked by BOT for an inappropriate name");
+                        builder.Color = Color.DarkOrange;
+                        await badUser.SendMessageAsync("", false, builder.Build());
+
+                        await badUser.KickAsync($"{badUser.Username}#{badUser.Discriminator} -> Inappropriate Name");
+
+                        await channel.SendMessageAsync($"Got it! I kicked {badUser.Username}#{badUser.Discriminator}!");
+                    }
+                    catch (Exception ex)
+                    {
+                        await channel.SendMessageAsync("Unable to kick user. Do I have the permissions?");
+                        Log.Error("Unable to kick user for bad name: " + ex);
+                    }
+                    messageDic.Remove(msg.Id);
+                    return;
+                }
+                if (reaction.Emote.Name.Equals(removeEmoji.Name))
+                {
+                    try
+                    {
+                        await badUser.Guild.GetUser(badUser.Id).ModifyAsync(user => user.Nickname = "");
+                        await channel.SendMessageAsync($"Got it! I changed {badUser.Username}#{badUser.Discriminator}'s nickname!");
+                    }
+                    catch (Exception ex)
+                    {
+                        await channel.SendMessageAsync("Unable to remove their nickname. Do I have the permissions?");
+                        Log.Error("Unable to remove nickname for bad name: " + ex);
+                    }
+                    messageDic.Remove(msg.Id);
+                    return;
+                }
+                else
+                    return;
```

Comment:
This else case is unnecessary -- the method will return by default after reaching the end.

Reply:
I can remove the else, but I would like to leave a return statement at the end, just incase of future expansion